apps/query-eval/queryeval/driver.py

```diff
@@ -476,6 +551,13 @@ def eval_all(self):
 
     def run(self):
         """Run all stages."""
+        # Define scorers
+        bleu_scorer = BleuScore()
```

Comment:
can you move this to the __init__ method? let's people configure these easily too then

Reply:
Yeah no idea how I missed that. Thanks!